modules/rbac-system-roles.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-role-based-access-control.adoc
+:_module-type: CONCEPT
+[id="rbac-system-roles_{context}"]
+= System roles
+
+[role="_abstract"]
+{product-title} includes some default system roles that you can apply on users.
+You can also create custom roles as required.
+
+|===
```

Comment:
Not critical, but I'd suggest adjusting the column widths to make the description column wider.